src/App.vue

```diff
@@ -550,37 +554,37 @@ export default {
       this.storeConfigObject(business.legalType)
     },
 
-    /** Verifies and stores an IA's data. */
-    storeIncorpApp (ia: any): void {
-      const filing = ia?.filing
-      if (!filing || !filing.business || !filing.header || !filing.incorporationApplication) {
-        throw new Error('Invalid IA filing')
+    /** Verifies and stores a draft applications data. */
+    storeDraftApp (application: any): void {
+      const filing = application?.filing
+      const filingName = filing.header?.name as FilingTypes
+      if (!filing || !filing.business || !filing.header || !filingName) {
+        throw new Error(`Invalid ${filingName} filing`)
       }
 
       const identifier = filing.business.identifier as string
       if (!identifier) {
-        throw new Error('Invalid IA filing - business identifier')
+        throw new Error(`Invalid ${filingName} filing - business identifier`)
       }
 
-      const name = filing.header.name as FilingTypes
-      if (name !== FilingTypes.INCORPORATION_APPLICATION) {
-        throw new Error('Invalid IA filing - filing name')
+      if (![FilingTypes.INCORPORATION_APPLICATION, FilingTypes.REGISTRATION].includes(filingName)) {
+        throw new Error(`Invalid ${filingName} filing - filing name`)
       }
 
       const status = filing.header.status as FilingStatus
       if (!status) {
-        throw new Error('Invalid IA filing - filing status')
+        throw new Error(`Invalid ${filingName} filing - filing status`)
       }
 
-      const nameRequest = filing.incorporationApplication.nameRequest
+      const nameRequest = filing[filing.header.name].nameRequest
```

Comment:
Did the linter complain about using `filingName` here instead (due to type)?

Reply:
mm i don't think so. Will double check and can update 👍

Reply:
Updated!